code/modules/mob/living/carbon/human/npcs.dm

```diff
@@ -21,3 +21,26 @@
 	real_name = name
 	resize = 0.8
 	update_transform()
+
+/mob/living/carbon/human/monkey/infect_with_monkey_virus(mob/user)
+	if(!mind && !client)
+		INVOKE_ASYNC(src, PROC_REF(spawn_new_monkey))
+	if(!HAS_TRAIT(src, TRAIT_HAS_MONKEY_VIRUS))
+		ADD_TRAIT(src, TRAIT_HAS_MONKEY_VIRUS, "Monkey virus")
+		to_chat(src, "<span class='userdanger'>You are an infected monkey! The touch of your hairy hands can infect anyone, and using your harm intent bite on people will cause them to turn to your side quicker.</span>")
+
+/mob/living/carbon/human/monkey/proc/spawn_new_monkey()
+	var/list/candidates = SSghost_spawns.poll_candidates("Do you want to play as a monkey?", source = /mob/living/carbon/human/monkey)
+	var/mob/C = null
+
+	if(!length(candidates))
+		return // no monkey :(
+	C = pick(candidates)
+	if(QDELETED(C))
+		return // monkey left :(
+	key = C.key
+	mind.name =	name
+	mind.assigned_role = ROLE_MONKEY
+	mind.special_role = ROLE_MONKEY
+	ADD_TRAIT(src, TRAIT_HAS_MONKEY_VIRUS, "Monkey virus")
+	to_chat(src, "<span class='userdanger'>You are an infected monkey! The touch of your hairy hands can infect anyone, and using your harm intent bite on people will cause them to turn to your side quicker.</span>")
```

Comment:
smh

```suggestion
	to_chat(src, "<span class='userdanger'>You are an infected monkey! The touch of your hairy hands can infect anyone, and using your harm intent bite on people will cause them to turn to your side quicker.</span>")

```

Reply:
I hastily copy and pasted this from my actual branch lmao